doorkeeper-openid_connect.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec-rails'
   spec.add_development_dependency 'factory_girl'
   spec.add_development_dependency 'sqlite3'
-  spec.add_development_dependency 'pry-byebug'
+  spec.add_development_dependency 'pry-byebug', '3.4.3'
+  spec.add_development_dependency 'byebug', '~> 9.0'
```

Comment:
Please either remove the version constraints, or add a comment explaining why they're necessary. Also `byebug` is already required by `pry-byebug` so maybe we can remove it?